core/audit/audit.go

```diff
@@ -52,50 +53,72 @@ func Log(e JournalEntry) {
 	if e.Entity == nil {
 		printAudit(
 			e.CorrelationId,
+			"nil",
 			e.Method, e.Url, e.Svid, string(e.Event),
 		)
 	}
 
 	switch v := e.Entity.(type) {
+	case reqres.SecretDeleteRequest:
```

Comment:
Fixes "unknown entity" warning that we get at times.
Especially when deleting things.